### PR TITLE
Navigate to composer browser window when the composer starts

### PR DESCRIPTION
**Pull request by NatashaWso2**
Works with : Linux, Windows, Mac
Fixes https://github.com/ballerinalang/composer/issues/2245

### CODE REVIEW

File: modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/app/StartBrowser.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+/**
+ * Starting the default browser once the composer starts
+ */
+public class StartBrowser {
+
+    public static void startInDefaultBrowser(String url) throws IOException {
+        Locale locale = Locale.ENGLISH;
+        String os = System.getProperty("os.name").toLowerCase(locale);
+        Runtime rt = Runtime.getRuntime();
+            if (os.indexOf("win") >= 0) {
```
Comment (pahans) [inline modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/app/StartBrowser.java]: you can use LaunchUtils.isWindows method to check this https://github.com/ballerinalang/composer/blob/master/modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/launcher/util/LaunchUtils.java#L32

File: modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/app/StartBrowser.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2016, WSO2 Inc. (http://wso2.com) All Rights Reserved.
```
Comment (hemikak) [inline modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/app/StartBrowser.java]: Year is wrong.

Comment (NatashaWso2) [reply modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/app/StartBrowser.java]: Fixed with fbbb851

File: modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/app/StartBrowser.java (hunk 1 above)
Comment (NatashaWso2) [reply modules/services/workspace-service/src/main/java/org/ballerinalang/composer/service/workspace/app/StartBrowser.java]: Fixed with fbbb851

**Pull request merged**